fenix/detekt-baseline.xml

```diff
@@ -893,6 +893,7 @@
     <ID>UndocumentedPublicFunction:TabLayout.kt$fun TabLayout.isSyncedModeSelected(): Boolean</ID>
     <ID>UndocumentedPublicFunction:TabLayoutMediator.kt$TabLayoutMediator$fun selectTabAtPosition(position: Int)</ID>
     <ID>UndocumentedPublicFunction:TabPreview.kt$TabPreview$fun loadPreviewThumbnail(thumbnailId: String)</ID>
+    <ID>UndocumentedPublicFunction:TabPreview.kt$TabPreview$fun setTabCount(count: Int)</ID>
```

Comment:
This can also be removed after the above. As an aside, we would always prefer to have comments instead of suppressing this lint check.